sql/updates/latest-dev.sql

```diff
@@ -17,3 +17,52 @@ DROP FUNCTION IF EXISTS _timescaledb_functions.invalidation_process_hypertable_l
 DROP FUNCTION IF EXISTS _timescaledb_functions.invalidation_process_hypertable_log(integer,integer,regtype,integer[],bigint[],bigint[],text[]);
 DROP FUNCTION IF EXISTS _timescaledb_functions.hypertable_invalidation_log_delete(integer);
 
+-- Cleanup metadata for deleted chunks
+CREATE UNLOGGED TABLE _timescaledb_catalog._chunks_remove AS
```

Comment:
Nit: Consider prefixing the table with `_tmp`

Reply:
Turned it into a PL/pgSQL function!